src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -64,14 +60,18 @@ public NsUser getWriter() {
         return writer;
     }
 
-    public String getContents() {
-        return contents;
-    }
 
     public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public void validateAnswerOwner(NsUser loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
    public void delete(NsUser loginUser) throws CannotDeleteException {
```
삭제 가능 여부를 확인하는 것이 핵심이 아니라 삭제가 핵심이지 않을까?